src/ccl_utils.c

```diff
@@ -30,6 +30,56 @@ double * ccl_linear_spacing(double xmin, double xmax, int N)
   return x;
 }
 
+/* ------- ROUTINE: ccl_linlog spacing ------
+ * INPUTS: [xminlog,xmax] of the interval to be divided in bins
+ *         xmin when linear spacing starts
+ *         Nlog number of logarithmically spaced bins 
+ *         Nlin number of linearly spaced bins 
+ * OUTPUT: bin edges in range [xminlog,xmax]
+ * */
+
+double * ccl_linlog_spacing(double xminlog, double xmin, double xmax, int Nlog, int Nlin)
```

Comment:
The linlog spacing routine exits for some values of the parameters. Have you checked that CCL exits correctly and prints the right message for these cases both in C and python?

Reply:
There was indeed one case where CCL did not exit correctly. I have corrected the error in the linlog function in src/ccl_utils.c and checked that CCL exists correctly and prints the right message for all cases in C. @ damonge: What would you suggest to test whether CCL also exists correctly for all cases in python?

Reply:
@damonge : What would you suggest to test whether CCL also exists correctly for all cases in python?

Reply:
@elisachisari : this function is never called from python (the same way ccl_linear_spacing wasn't before), so the errors have to be caught at the C level. We have verified that we always check the output within C, and that if the function has failed, the error gets delivered to python.